xml/cl.xml

```diff
@@ -1779,7 +1783,8 @@ server's OpenCL/api-docs repository.
         <enum value="0x12B4"        name="CL_IMAGE_REQUIREMENTS_MAX_HEIGHT_EXT"/>
         <enum value="0x12B5"        name="CL_IMAGE_REQUIREMENTS_MAX_DEPTH_EXT"/>
         <enum value="0x12B6"        name="CL_IMAGE_REQUIREMENTS_MAX_ARRAY_SIZE_EXT"/>
-            <unused start="0x12B7" end="0x1FFF" comment="Reserved for core API tokens"/>
+        <enum value="0x12B7"        name="CL_MUTABLE_DISPATCH_PROMISES_KHR"/>
```

Comment:
The spec says `CL_MUTABLE_DISPATCH_ASSERTS_KHR` not `CL_MUTABLE_DISPATCH_PROMISES_KHR`